src/nvim/ex_docmd.c

```diff
@@ -9224,6 +9224,24 @@ static void ex_filetype(exarg_T *eap)
     EMSG2(_(e_invarg2), arg);
 }
 
+/// Same as :filetype plugin indent enable
+/// Updates the state used for :filetype without args.
+void force_enable_filetype(void)
```

Comment:
"force" isn't a good name here because it actually checks if the actions were already performed and skips them if so...